src/observable/fromEvent.ts

```diff
@@ -1,40 +1,60 @@
 import {Observable} from '../Observable';
+import {isEventTarget} from '../util/isEventTarget';
+import {isHTMLCollection} from '../util/isHTMLCollection';
+import {isNodeList} from '../util/isNodeList';
 import {tryCatch} from '../util/tryCatch';
 import {errorObject} from '../util/errorObject';
 import {Subscription} from '../Subscription';
 import {Subscriber} from '../Subscriber';
 
+export type EventEmmitterLike1 = {
```

Comment:
to @kwonoj's point of naming, I don't have much better names tbh.

`addListener`/`removeListener` is generally associated with NodeJS and it's `EventEmitter`.
`on`/`off` are generally associated with JQuery.

So only thing I can think of is something like `EventEmitterLike` and `JQueryEventEmitterLike`.